SmartDeviceLink/SDLLockScreenStatusManager.m

```diff
@@ -1,103 +0,0 @@
-//
-//  SDLLockScreenManager.m
-//  SmartDeviceLink
-//
-
-#import "SDLLockScreenStatusManager.h"
-
-#import "SDLLockScreenStatus.h"
-#import "SDLOnLockScreenStatus.h"
-
-NS_ASSUME_NONNULL_BEGIN
-
-@interface SDLLockScreenStatusManager ()
-
-@property (assign, nonatomic) BOOL haveDriverDistractionStatus;
-
-@end
-
-
-@implementation SDLLockScreenStatusManager
-
-#pragma mark - Lifecycle
-
-- (instancetype)init {
-    self = [super init];
-    if (self) {
-        _userSelected = NO;
-        _driverDistracted = NO;
-        _haveDriverDistractionStatus = NO;
-    }
-    return self;
-}
-
-
-#pragma mark - Getters / Setters
-#pragma mark Custom setters
-
-- (void)setDriverDistracted:(BOOL)driverDistracted {
-    _driverDistracted = driverDistracted;
-    _haveDriverDistractionStatus = YES;
-}
-
-- (void)setHmiLevel:(nullable SDLHMILevel)hmiLevel {
-    if (_hmiLevel != hmiLevel) {
-        _hmiLevel = hmiLevel;
-    }
-
-    if ([hmiLevel isEqualToEnum:SDLHMILevelFull] || [hmiLevel isEqualToEnum:SDLHMILevelLimited]) {
-        self.userSelected = YES;
-    } else if ([hmiLevel isEqualToEnum:SDLHMILevelNone]) {
-        self.userSelected = NO;
-    }
-}
-
-
-#pragma mark Custom Getters
-
-- (SDLOnLockScreenStatus *)lockScreenStatusNotification {
```

Comment:
Thanks for removing this, but it doesn't appear that the logic here has been moved to the lock screen manager. The lock screen doesn't even work as a result. The logic managed by this manager needs to be moved to the `SDLLockScreenManager` and `SDLOnLockScreenStatus` can be deleted (it's not an RPC).